File: Dataset/Leetcode/train/58/13.cpp
 class Solution {
public:
    int XXX(string s) {
        int n = s.size();
        int res = 0;
        for(int i = n-1; i >= 0; i--){
            if(s.at(i)==' ') continue;
            else {
                while(i >=0 && s.at(i) != ' '){
                    res++;
                    i--;
                }
                return res;
            }
        }
        return 0;
    }
};

File: Dataset/Leetcode/train/55/13.java
class Solution {
    public boolean XXX(int[] nums) {
        boolean res = true;
        int n = 1;
        for(int i = nums.length-1;i>0;i--){
            if(nums[i-1]>=n){
                res = true;
                n=1;
            }else{
                n++;
                res = false;
            }
        }
        return res;
    }
}

